src/services/balancer/subgraph/entities/pools/index.ts

```diff
@@ -229,6 +232,10 @@ export default class Pools {
     }
   }
 
+  private isNewPool(pool: Pool): boolean {
+    return differenceInWeeks(Date.now(), pool.createTime * 1000) < 1;
```

Comment:
Maybe use our named constants for time from here: https://github.com/balancer-labs/frontend-v2/blob/develop/src/composables/useTime.ts#L1 ?
```suggestion
    return differenceInWeeks(Date.now(), pool.createTime * oneSecondInMs) < 1;
```